generated/proto/policy.proto

```diff
@@ -34,3 +34,12 @@ message Policy {
 	Resolution resolution = 1;
 	Retention retention = 2;
 }
+
+message CompressedPolicy {
+	Policy policy = 1;
+	int64 id = 2;
+}
+
+message ActivePolicies {
```

Comment:
My thinking was that we would delay exposing policies in the UI until after a reasonable amount of time has passed from when we updated the policies in kv that we can be confident the updated set of policies has propagated to all of the collectors. 

I can see us adding a cutover time field here so that the UI would know when to expose the field to users. I don't think it would be very helpful to use it in the collector, I think checking a Policy's cutover time every time we look up its ID might introduce unnecessary overhead.